README.md

```diff
@@ -1,5 +1,15 @@
 # docker-alpine-node-flyway
-Minimal docker image based on node:8-alpine and is able to launch flyway
+
+## Dockerfile
+Minimal docker image based on node:8-alpine and is able to launch flyway.
+
+Addons:
+- openssl
+- openjdk8 (needed for flyway)
+- **flyway 5.2.1**
+
+## Dockerfile-node10
+Minimal docker image based on node:10-alpine and is able to launch flyway.
 
 Addons:
 - openssl
```

Comment:
Minor: looks like it contains openjdk8 as well like node:8-alpine version, so it needs to be written here as well